build.gradle

```diff
@@ -59,11 +59,13 @@ dependencies {
     // Terra deps
     implementation group: 'bio.terra', name: 'stairway', version: '0.0.52-SNAPSHOT'
     implementation group: "bio.terra", name: "terra-common-lib", version: "0.0.46-SNAPSHOT"
+
+    implementation platform('bio.terra.cloud-resource-lib:platform:0.1.0')
     implementation group: 'bio.terra.cloud-resource-lib', name: 'cloud-resource-schema'
```

Comment:
we define a `cloud-resource-schema` version above to  pull the openapi definitions for generating the janitor client. While we probably want to eventually get rid of that, for now should we no longer define the version on line 35 and get it from the platform?

Reply:
Ah yes, good catch, thanks.  I should have noticed that it didn't have a local version specified.